SurfaceDevCenterManager/DevCenterApi/DevCenterHandler.cs

```diff
@@ -129,7 +129,7 @@ private string GetDevCenterBaseUrl()
                     }
 
                     // reterr can be null when there is HTTP error
-                    if (reterr == null)
+                    if (reterr == null || reterr.HttpErrorCode == 0)
```

Comment:
Just confirming, is HttpErrorCode == 0 when there is an error? Or does this need to be !=

Reply:
Yes, the codes are the one from the list.  Sometimes I got a zero so I added the check